test/InjectorTest.php

```diff
@@ -341,4 +343,127 @@ public function testComplexDeepDependencyConfiguration()
         $this->assertSame($expected1, $result->result->result->optionalResult);
         $this->assertSame($expected2, $result->result2->result->optionalResult);
     }
+
+    public function testCreateInstanceWithoutUnknownClassThrowsException()
+    {
+        $this->expectException(Exception\ClassNotFoundException::class);
+        (new Injector())->create('Unknown.Alias.Should.Fail');
+    }
+
+    public function testKnownButInexistentClassThrowsException()
+    {
+        $definition = $this->getMockBuilder(DefinitionInterface::class)
+                           ->getMockForAbstractClass();
+
+        $definition->expects($this->any())
+                   ->method('hasClass')
+                   ->willReturn(true);
```

Comment:
On this call and the one above, indent only one extra-level; do not align with the initial line.